### PR TITLE
Feature/vtn 10191 mdp page header icon order

### PR DESCRIPTION
**Pull request by ChrisPelletier**
https://steel-ventures.atlassian.net/projects/VTN/issues/VTN-10200
https://steel-ventures.atlassian.net/projects/VTN/issues/VTN-10191
https://steel-ventures.atlassian.net/projects/VTN/issues/VTN-10312

Moved download button out from under the more menu in the header of MDP (VTN-10191)

Updated the TranscriptEngineOutput and FaceEngineOutput smart components to handle their own edit state and moved the edit button from the MDP header and into each of the components. This required some of the components and redux logic to be rewritten to accommodate. However this should allow us to get closer to making TranscriptEngineOutput and FaceEngineOutput smart components capable of being their own standalone components. (VTN-10200)

Disable engine select when in edit mode (VTN-10312)

### CODE REVIEW

File: packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js [hunk 1]
```diff
@@ -211,19 +224,60 @@ const reducer = createReducer(defaultState, {
...
+  [SAVE_FACE_EDITS](state) {
+    return {
+      ...state,
+      savingFaceEdits: true
+    };
+  },
+  [SAVE_FACE_EDITS_SUCCESS](state) {
+    return {
+      ...state,
+      error: null,
```
Comment (AlexOVeritone) [inline packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js]: You should declare 'error: null' in the default state.

File: packages/veritone-widgets/src/redux/modules/mediaDetails/transcriptWidget/saga.js [hunk 2]
```diff
@@ -118,6 +126,18 @@ function* watchDiscardUnsavedChanges() {
   });
 }
 
+function* watchMediaDetailCancelEdit() {
+  yield takeLatest([CANCEL_EDIT], function*() {
+    const pendingUserEdits = yield select(TranscriptRedux.hasUserEdits);
+    console.log('transcript watchMediaDetailCancelEdit', pendingUserEdits);
```
Comment (AlexOVeritone) [inline packages/veritone-widgets/src/redux/modules/mediaDetails/transcriptWidget/saga.js]: remove this log and below with if statement

File: packages/veritone-widgets/src/widgets/TranscriptEngineOutput/index.js [hunk 3]
```diff
@@ -241,36 +305,95 @@ export default class TranscriptEngineOutputContainer extends Component {
...
     ]);
 
-    const alertTitle = 'Unsaved Transcript Changes';
-    const alertDescription =
-      'This action will reset your changes to the transcript.';
-    const cancelButtonLabel = 'Cancel';
-    const approveButtonLabel = 'Continue';
+    let alertTitle = 'Unsaved Transcript Changes';
+    let alertDescription =
+      'This action will reset y;our changes to the transcript.';
```
Comment (AlexOVeritone) [inline packages/veritone-widgets/src/widgets/TranscriptEngineOutput/index.js]: y;our / your

File: packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js (hunk 1 above)
Comment (ChrisPelletier) [reply packages/veritone-widgets/src/redux/modules/mediaDetails/faceEngineOutput/index.js]: done

**Pull request merged**